README.md

```diff
@@ -103,10 +105,7 @@ The Lob constructor accepts an `options` object which may contain one or more of
 
 We've provided various examples for you to try out [here](https://github.com/lob/lob-node/tree/master/examples).
```

Comment:
I like this view better because you can see the README and the example directories at the same time.